### ISSUE TITLE
Rfracs from bias steps should be plotted and saved after bias steps

### ISSUE THREAD

**Issue by jlashner** [simonsobs/sodetlib#343]
Rfracs from bias steps should be plotted and saved after bias steps

**Comment by jseibert575** [comment]
Maybe I’m not understanding why, but isn’t this already done? Not plotted, but they are effectively saved in the bias step analysis files because the calculated resistance is saved and you have the normal resistance from IV entry as well, right?

**Comment by jlashner** [comment]
Yeah they're definitely saved in the analysis file, so we'll always be able to recover it. This mainly came up because Caleb's working on a plot-viewer for plots transferred during operation at the site, and I think Rfracs in operation will always be a great thing to look at to debug.
